Snakefile

```diff
@@ -257,8 +248,7 @@ rule preseg:
 rule ploidetect:
     """Runs Ploidetect"""
     input:
-        rules.preseg.output,
-	    check_docker()
```

Comment:
Because the output of install_ploidetect is no longer the input of any job, the rule would never run anymore. Why was this change made along with removing the check_docker function?

Reply:
The install_ploidetect rule is still around at the moment, when _not_ using docker:
```
if not config["use_docker"]:
    # This ensures that install_ploidetect is run first
    ruleorder: install_ploidetect > ploidetect
```
 - Used this in a conda env to find that the one R module was missing from the conda config.

Reply:
the ruleorder keyword is for resolving ambiguous rules where two rules make the same outputs. For a rule to be run in Snakemake, its output needs to be an input for a rule whose output is, in turn required by another (and so on until rule all). Cloning the branch and dry-running confirms that install_ploidetect is not run.

Reply:
I see - the ruleorder didn't actually do what I thought it did, the order of execution was random.
I just made the rule exist or not based on the `if not config["use_docker"]:`